tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -3660,15 +3661,6 @@ public function provideFixMultiLineStructuresCases()
             [
                 '<?php
     class Foo
-    {
```

Comment:
Why did you removed perfectly fine test?

Reply:
I did not :) - I only corrected the expected output and then the input was matching, thus the expected output is displayed as removed instead of the 2nd code block with the input.

Reply:
The removed expected output is correct, for function:
```
    public static function bar(
            $baz,
            $boo
        )
        {
        }
````
we want `)` and `{` to be change to be in the same line.

Reply:
No, the whole PR is not to have these on the same line if `position_after_functions_and_oop_constructs` = `next`.

Reply:
Yes, this test here is for a reason - if you have `position_after_functions_and_oop_constructs` with `next`, BUT you have multiline arguments then it is an exception - you can see how many other files got changed to wrong style.

Reply:
Yes, this was a bug.

Reply:
I'd say it wasn't. If it was I'd say someone would notice it few years ago.

Reply:
I'm unresolving this thread.
@mvorisek , we do appreciate your effort into making the PHP CS Fixer better! Yet, please keep close attention to the hints you receive by the core team.

PSR is asking to keep `) {` on same line when the argument list is split across multiple lines
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5260#issuecomment-822070978